libs/blocks/merch/merch.js

```diff
@@ -1,57 +1,151 @@
-import { loadScript, getConfig, createTag } from '../../utils/utils.js';
+import {
+  loadScript,
+  getConfig,
+  createTag,
+} from '../../utils/utils.js';
 
-const { miloLibs, codeRoot, env } = getConfig();
-const base = miloLibs || codeRoot;
+const wcs = { apiKey: 'wcms-commerce-ims-ro-user-cc' };
+const envProd = 'prod';
+const ctaPrefix = /^CTA +/;
 
-function buildButton(a, osi) {
-  if (!a) return null;
+let initialized = false;
+
+// supported languages for the price literals
+const supportedLanguages = [
+  'ar', 'bg', 'cs', 'da', 'de', 'en', 'es', 'et', 'fi', 'fr', 'he', 'hu', 'it', 'ja', 'ko',
+  'lt', 'lv', 'nb', 'nl', 'pl', 'pt', 'ro', 'ru', 'sk', 'sl', 'sv', 'tr', 'uk', 'zh_CN', 'zh_TW',
+];
+
+/**
+ * Removes undefined properties from an object
+ * @param {*} target Object with potentially undefined properties
+ * @returns new Object without undefined properties
+ */
+export function omitUndefined(target) {
+  if (target != null) {
+    Object.entries(target).forEach(([key, value]) => {
+      if (value == null) delete target[key];
+    });
+  }
+  return target;
+}
+
+const getTacocatEnv = (envName, ietf) => {
+  const scriptUrl = (envName === envProd
```

Comment:
envProd can be moved inside this method. No reason to have it global.

Reply:
this is to reach 100% test coverage. Otherwise it is not easy to get this method called with `env = 'prod'`

Reply:
ok 👍